packages/modals/src/styled/StyledTooltipModalBackdrop.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { DEFAULT_THEME, retrieveComponentStyles } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'modals.tooltip_modal.backdrop';
+
+/**
+ * 1. Smooth iOS scrolling.
+ */
+export const StyledTooltipModalBackdrop = styled.div.attrs({
```

Comment:
Looks like https://github.com/zendeskgarden/react-components/blob/main/packages/modals/src/styled/StyledBackdrop.ts could be factored to extend this styled component.

Reply:
Same comment as https://github.com/zendeskgarden/react-components/pull/828#discussion_r469426145